### ISSUE TITLE
Usability Metadata Editor: Show multiple thumbnail sizes

### ISSUE THREAD

**Issue by BartChris** [kitodo/kitodo-production#5154]
Right now the metadata edtior has only two sizes: thumbnails and full size. The thumbnails are very small and the full size view is not usable to get an overview over the whole document. While structuring a document it would help if there was the option to increase the display size of thumbnails in order to see more details of the images since . 

Maybe provide three different display sizes for thumbnails. This should already help even if there is still only one physical thumbnail on the server.

**Comment by matthias-ronge** [comment]
To increase thumbnail size, you need to change the px size under project settings → folder → generate image. If you want to stay compliant to DFG profile, create another folder with larger thumbnails and select that folder for preview in the project settings. Do not export that folder, but export the one with the 150 px width images.

Important: You will _need to_ pre-generate the larger thumbnails. Just displaying the existing thumbnails larger would only give you smeared images, you won’t see more detail.

Support of an image server which generates the images on the fly is an outstanding wish … see #3371

**Comment by BartChris** [comment]
You are of course correct that enlarged images require higher resolution. The **display** size in Kitodo production however does not change, even if one uses the full size images as thumbnails. So my idea would be to offer a button to change the display size and it would be of course in the responsibility of the institution to provide large enough thumbnails to display something usable here.

**Comment by solth** [comment]
Maybe the available resolution options in the gallery could be linked directly to the available filegroups in the projects settings? So if one thumbnail folder is configured with image width 150px and another one with 200px, the metadata editor would offer buttons for thumbnail images in the gallery with the corresponding sizes.

**Comment by matthias-ronge** [comment]
The thumbnail size should be 1:1 the size of the image used for preview, I suggest.

**Comment by BartChris** [comment]
I agree that having the selectable sizes in the editor relate to the generated image derivatives is a clean solution. It would however mean a lot of overhead generating multiple thumbnail derivivatives. 
Let's say i would generate a 350px wide image, which could be reused for smaller thumbnail sizes as well. The browser can just downscale the display size. One could also generate a 150px derivative for the DFG if this is necessary or generate that out of Kitodo. But i do not see a reason to generate mutiple thumbnails just for display purposes in Kitodo.
